lib/telnet.c

```diff
@@ -81,8 +81,6 @@
   } WHILE_FALSE
 
 #define  CURL_SB_GET(x) ((*x->subpointer++)&0xff)
-#define  CURL_SB_PEEK(x)   ((*x->subpointer)&0xff)
-#define  CURL_SB_EOF(x) (x->subpointer >= x->subend)
```

Comment:
I'd like to see the warnings fixed yes, so having them commented out is probably a fair compromise.